docs/documentation/full-text/highlighting.mdx

```diff
@@ -46,3 +46,24 @@ FROM mock_items
 WHERE description @@@ 'shoes'
 LIMIT 5;
 ```
+
+## Snippet Positions
+
+`paradedb.snippet_positions(<column>)` is used to return the start position and length of the highlighted snippet.
+
+```sql
+SELECT id, paradedb.snippet_positions(description)
```

Comment:
This should add the start/end right?

```
SELECT id, paradedb.snippet_positions(description, start_tag => '<i>', end_tag => '</i>')
```

As per your example in the PR body

Reply:
The `start_tag` and `end_tag` are unnecessary for the `snippet_positions`, so I have removed them.